src/Quartz/CronExpression.cs

```diff
@@ -208,12 +208,21 @@ namespace Quartz
     /// <author>Contributions from Mads Henderson</author>
     /// <author>Refactoring from CronTrigger to CronExpression by Aaron Craven</author>
     /// <author>Marko Lahma (.NET)</author>
+#if BINARY_SERIALIZATION
     [Serializable]
-    public class CronExpression : ICloneable, IDeserializationCallback
+#endif // BINARY_SERIALIZATION
+    [DataContract]
+    public class CronExpression : object
+#if ICLONEABLE
```

Comment:
If we don't need to implement ICloneable on desktop .NET then dropping it would make the source a little cleaner.